docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -2498,20 +2516,182 @@ JSON file that specifies:
             "enabled": true,
-            "gas_limit": "12345654321"
+            "gas_limit": "35000000"
           }
-        }
+        },
+        "0xa99a76ed7796f7be22d5b7e85deeb7c5677e88e511e0b337618f8c4eb61349b4bf2d153f649f7b53359fe8b94a38e44c": {
+          "builder": {
+            "enabled": true
+          }
+        },
       },
       "default_config": {
-        "fee_recipient": "0x6e35733c5af9B61374A128e6F85f553aF09ff89A"
+        "fee_recipient": "0x6e35733c5af9B61374A128e6F85f553aF09ff89A",
         "builder": {
           "enabled": false,
-          "gas_limit": "12345654321"
+          "gas_limit": "25000000"
+        }
+      }
+    }
+    ```
+
+    In this example, validator `0xa0578...` is configured as:
+    
+    ```json
+    "fee_recipient": "0x50155530FCE8a85ec7055A5F8b2bE214B3DaeFd3",
+    "builder": {
+      "enabled": true,
+      "gas_limit": "35000000"
+    }
+    ```
+    
+    Validator `0xa99a7...` is configured as:
+    
+    ```json
+    "fee_recipient": "0x6e35733c5af9B61374A128e6F85f553aF09ff89A",
+    "builder": {
+      "enabled": true,
+      "gas_limit": "25000000"
+    }
+    ```
+    
+    All other validators are configured as:
+    
+    ```json
+    "fee_recipient": "0x6e35733c5af9B61374A128e6F85f553aF09ff89A",
+    "builder": {
+      "enabled": false,
+      "gas_limit": "25000000"
+    }
+    ```
+
+The following is an example in conjunction with CLI argument.
+
+??? example "`proposerConfigAndCLI.json`"
+
+    ```json
+    {
+      "proposer_config": {
+        "0xa057816155ad77931185101128655c0191bd0214c201ca48ed887f6c4c6adf334070efcd75140eada5ac83a92506dd7a": {
+          "fee_recipient": "0x50155530FCE8a85ec7055A5F8b2bE214B3DaeFd3",
+          "builder": {
+            "gas_limit": "35000000"
+          }
+        },
+        "0xa99a76ed7796f7be22d5b7e85deeb7c5677e88e511e0b337618f8c4eb61349b4bf2d153f649f7b53359fe8b94a38e44c": {
+          "builder": {
+            "enabled": false
+          }
+        },
+      },
+      "default_config": {
+        "fee_recipient": "0x6e35733c5af9B61374A128e6F85f553aF09ff89A",
+      }
+    }
+    ```
+
+    * If `--validators-builder-registration-default-enabled` is specified in CLI
+
+    Validator `0xa0578...` is configured as:
+    
+    ```json
+    "fee_recipient": "0x50155530FCE8a85ec7055A5F8b2bE214B3DaeFd3",
+    "builder": {
+      "enabled": true,
+      "gas_limit": "35000000"
+    }
+    ```
+
+    Validator `0xa99a7...` is configured as:
+
+    ```json
+    "fee_recipient": "0x6e35733c5af9B61374A128e6F85f553aF09ff89A"
+    "builder": {
+      "enabled": false,
+      "gas_limit": "30000000"
+    }
+    ```
+
+    All other validators are configured as:
+    
+    ```json
+    "fee_recipient": "0x6e35733c5af9B61374A128e6F85f553aF09ff89A",
+    "builder": {
+      "enabled": true,
+      "gas_limit": "30000000"
+    }
+    ```
+
+    * If `--validators-builder-registration-default-enabled` is NOT specified (or set to false) in CLI
```

Comment:
@alexandratran the UI isn't that good for those IMO. Don't know if you have other ways to represent these two "* if .." contitions, otherwise we can merge.